### ISSUE TITLE
Fun

### ISSUE THREAD

**Issue by vipnvap** [mdbesten/what-could-go-wrong-llm#18]
The gameplay invites the palyers to brainstorm together and challenge each other's ideas so it is a very collaborative and engaging game. But it could be more fun considering than the debate can sometimes adress an issue that is not clear enough or on which the players are not instructed enough to have a precise opinion.

**Comment by mdbesten** [comment]
Can you identify issues that need clarification or suggest mechanisms to ensure that players can instruct themselves prior to or during the game?

**Comment by vipnvap** [comment]
What could sometimes need clarification is the content of the black prompt cards. A mechanism that could ensure that the payers understand it or instruct themselves about it could be for the all the players to discuss the content of the prompt card to make sure everyone has the same understanding before proposing white answers cards. If players disagree on the interpretation, the cardzar could choose the interpretation. It could also be the occasion for an uninformed player to ask for explanation.

**Comment by mdbesten** [comment]
What kind of mechanism do you have in mind? A QR code/link to background information?

**Comment by vipnvap** [comment]
I think a QR code that links to background information is a good idea but it would take time to implement. Plus, I think it would be more fun for the payers to precise in the rules that in the case of a player being not sure of the definition of the prompt card, that the process to adopt would be : 
1. For this player to tell others 
2. a. If every other players have the same definition in mind : explain it to the player who did not know and continue the game. 
b. If the other players disagree on the right definition : the whole team debate on the right definition to adopt. 
3. If people disagree, the Card Czar decide it. 

I think it would be more : 
- fun 
- involving 
for all the players because it helps : 
1. everyone to better understand the prompt card, which makes the game smoother which make every players involved, 
2. people debate, which add fun to the game. 

This is for this matter that I pulled request on the READme during our second class.

**Comment by 54R7EURTIF** [comment]
Yes that would a great idea. I am working on a way to implement more fun and involvment in this game as well. Let's collaborate !! :)
